code/modules/mob/living/silicon/robot/life.dm

```diff
@@ -153,6 +153,8 @@
 	if(!is_component_functioning("actuator"))
 		src.Paralyse(3)
 
+	if(ticker && ticker.mode.name == "nations")

```

Comment:
This needs to be in living/Life() and ai/Life() only, not in robot, human, and ai.